lib/vanagon/platform/windows.rb

```diff
@@ -107,21 +107,22 @@ def generate_msi_package(project)
         target_dir = project.repo ? output_dir(project.repo) : output_dir
         cg_name = "ProductComponentGroup"
         dir_ref = "INSTALLDIR"
+        candle_flags =  "-dPlatform=#{@architecture} -arch #{@architecture} -ext WiXUtilExtension -ext WixUIExtension"
```

Comment:
I have created ticket [RE-6505](https://tickets.puppetlabs.com/browse/RE-6505) to pick up the issue of where the bill-of-materials goes. Have also created [RE-6506](https://tickets.puppetlabs.com/browse/RE-6506) to ensure that the project version is set correctly.